sui/src/benchmark/bench_types.rs

```diff
@@ -24,7 +22,7 @@ pub struct Benchmark {
     #[clap(long, default_value = "40000000", global = true)]
     pub recv_timeout_us: u64,
     /// Maximum size of datagrams received and sent (bytes)
-    #[clap(long, default_value = transport::DEFAULT_MAX_DATAGRAM_SIZE_STR, global = true)]
+    #[clap(long, default_value = "65000", global = true)]
```

Comment:
Where is this number from?
Can we make it a const?

Reply:
same comment as https://github.com/MystenLabs/sui/pull/1687#discussion_r862957573